app/views/groups/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container category-container">
+  <div class="d-flex justify-content-between navbar category-head">
+    <h3>All Categories</h3>
+    <%= link_to 'Sign out', destroy_user_session_path, class: 'signOut', method: :delete %>
+  </div>
+
+  <div class="categories dev">
+    <% @groups.each_with_index do |cat, index|%>
+      <%= link_to group_path(cat) do %>
+        <div class="category card mb-3 category-list">
+          <div class="row g-0">
+             <div class="col-md-4">
+             <%= image_tag(cat.icon, class: "img-fluid", alt: cat.name, style: "height: 100px; width:100px;") if cat.icon.attached? %>
+        </div>
+            <div class="col-md-8">
+              <div class="card-body text-dark">
+                <div class="d-flex justify-content-between">
+                  <h5 class="card-title"><%= cat.name %></h5>
+                <p class="card-text">Total: $<%= @totals[index] || 0 %></p>
+                </div>
+                <div>
```

Comment:
- [ ] as the previous reviewer pointed out, kindly keep to design guide i.e color, font and spacing, your texts have text decoration which is not on the design guide, also in the total, the text `total` is not in the design guide, so remove it
your design :point_down: 
![image](https://github.com/arnoldnekemiah/Budget-app/assets/66097371/eb35c23e-25b0-4342-b479-e259201efe79)
design guide :point_down: 
![image](https://github.com/arnoldnekemiah/Budget-app/assets/66097371/788cf65d-4845-4466-bea8-34d01d71e837)